### ISSUE TITLE
CSV file contents- change of approach

### ISSUE THREAD

**Issue by brotzmanmj** [episphere/biospecimen#261]
Currently the CSV file is not being generated in a way that aligns with the manifest of which tubes were actually shipped and received.  
Needs discussion and a broader change in approach, preferably to be deployed by end of July when pace of sample collections ramp up.

**Comment by wyattks** [comment]
@abhinavjonnada82 I shipped and received the remainder of the CXA000102 samples today.  The csv file was correct in the Mouthwash sample showed up only in the report run with yesterday's date (7/12/2022) and not today's date.  All the samples received today had the correct date associated with them.  The only issue I came across was that each tube was listed in the report run today (7/13/2022) 6 times.  I have attached both files.  Michelle and Stephanie tried running a report with today's date and got the same results as I did.

[CXA000102 7-13-2022 receipt.csv](https://github.com/episphere/biospecimen/files/9102734/CXA000102.7-13-2022.receipt.csv)


[CXA000102 7-12-2022 receipt.csv](https://github.com/episphere/biospecimen/files/9102735/CXA000102.7-12-2022.receipt.csv)
.
